x-pack/solutions/observability/test/api_integration_deployment_agnostic/apis/ai_assistant/complete/functions/title_conversation.spec.ts

```diff
@@ -76,9 +77,10 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
       });
 
       it('sends the correct system message to the LLM for the title', () => {
+        const systemMessage = getTitleSystemMessage(['observability'] as AssistantScope[]);
```

Comment:
nit: this might work as well:

```suggestion
        const systemMessage = getTitleSystemMessage(['observability' as const]);
```